site/using-cluster-operator.md

```diff
@@ -0,0 +1,790 @@
+
+**Example:**
+
+<pre class="terminal">
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: rabbitmqcluster-sample
+spec:
+  service:
+    type: LoadBalancer
+</pre>
+
+### <a name='service-annotations' class='anchor' href='#service-annotations'>Service Annotations</a>
+
+**Description:** Specify the Kubernetes Service annotations for the RabbitmqCluster Service. The Services created
+by the Cluster Operator will have these annotations.
+
+**Default Value:** N/A
+
+**Example:**
+
+<pre class="terminal">
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: rabbitmqcluster-sample
+spec:
+  service:
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+</pre>
+
+### <a name='persistence' class='anchor' href='#persistence'>Persistence</a>
+
+**Description:** Specify the persistence settings for the RabbitmqCluster Service.
+The available settings are:
+
+* `storageClassName`: The name of the Kubernetes StorageClass to use.
+    <p class="note">
+      <strong>Note:</strong> If your cluster does not have a default StorageClass, this property
+      must be set, otherwise RabbitMQ Pods will not be scheduled because they require a Persistent Volume.
+    </p>
+* `storage`: The capacity of the persistent volume, expressed as a Kubernetes resource quantity.
+
+**Default Values:**
+
+* `storageClassName`: Not set by default. If you do not set a value, the default `StorageClass` for the Kubernetes cluster is used.
+* `storage`: 10Gi
+
+To see the default `StorageClass`, run `kubectl get storageclasses`.
+
+**Example:**
+
+<pre class="terminal">
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: rabbitmqcluster-sample
+spec:
+   persistence:
+    storageClassName: fast
+    storage: 20Gi
+</pre>
+
+For more information about concepts mentioned above, see:
+
+<table class="nice">
+<col width="50%"></col>
+<col width="50%">
+	<th>Concept</th>
+	<th>More information in…</th>
+	<tr>
+		<td>StorageClass</td>
+		<td>The <a href="https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource">Kubernetes documentation</a></td>
+	</tr>
+	<tr>
+		<td>Persistent volume capacity</td>
+		<td>The <a href="https://kubernetes.io/docs/concepts/storage/persistent-volumes/#capacity">Kubernetes documentation</a></td>
+	</tr>
+	<tr>
+		<td>Kubernetes Resource Quantity</td>
+		<td>The <a href="https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#resource-quantities">Kubernetes Resource Model documentation</a> in GitHub</td>
+	</tr>
+    </col>
+ </table>
+
+### <a name='resource-reqs' class='anchor' href='#resource-reqs'>Resource Requirements</a>
+
+**Description:** Specify the resource requests and limits of the `RabbitmqCluster` Pods.
+
+CPU requirements must be in CPU units. Memory requirements must be in bytes.
+
+You must express both values as a Kubernetes resource quantity.
```

Comment:
We tend to use "X must be done" instead of "you must do X" in doc guides.

Reply:
I've made some changes locally to rewrite the "Using" guide using impersonal sentences. I will revisit the other guides too and make this change.